HTTP2-Core.package/H2Frame.class/instance/flagIsSet..st

```diff
@@ -0,0 +1,3 @@
+testing
+flagIsSet: aBitPosition
```

Comment:
isFlagSet ist vielleicht ein besserer Methodenname

Reply:
ich wäre einverstanden mit isFlagSet:

Reply:
Kapselung der flags in einer Flag-Klasse